testHelper/simulation.go

```diff
@@ -237,40 +256,52 @@ func TimeNow(s *state.State) {
 	fmt.Printf("%s:%d-:-%d Now %s of %s (remaining %s)\n", s.FactomNodeName, int(s.LLeaderHeight), s.CurrentMinute, now.Sub(startTime).String(), endTime.Sub(startTime).String(), endTime.Sub(now).String())
 }
 
+var statusMutes sync.Mutex
 var statusState *state.State
+var StatusEveryMinuteRunning bool
 
 // print the status for every minute for a state
 func StatusEveryMinute(s *state.State) {
-	if statusState == nil {
-		fmt.Fprintf(os.Stdout, "Printing status from %s\n", s.FactomNodeName)
-		statusState = s
-		go func() {
-			for {
-				s := statusState
-				if s != nil {
+	statusMutes.Lock()
+	statusState = s
+	statusMutes.Unlock()
+
+	if s == nil {
+		fmt.Fprintf(os.Stdout, "StatusEveryMinute OFF\n")
+	} else {
+		fmt.Fprintf(os.Stdout, "Printing status from %s", s.FactomNodeName)
+		if !StatusEveryMinuteRunning {
+			StatusEveryMinuteRunning = true
+			go func() {
+				for {
+					statusMutes.Lock()
+					s = statusState
+					statusMutes.Unlock()
+					if s == nil {
+						break // exit the routine
```

Comment:
@Emyrk wondering if changes here will fix the issue you're seeing w/ sim test

Reply:
Yes it will :+1: 

There is a similar fix in parchment. Either or will suffice :man_shrugging: